region/lowernorfair/west.json

```diff
@@ -94,18 +94,20 @@
                   "heatProof",
                   "canHeatRun"
                 ]},
-                {"and": [
-                  "SpaceJump",
-                  "canUsePowerBombs",
-                  {"cost": [
-                    {"heatFrames": 400}
-                  ]}
-                ]},
-                {"and": [
-                  "UsedAcidChozoStatue",
-                  "canUsePowerBombs",
-                  {"cost": [
-                    {"heatFrames": 450}
+                {"or": [
+                  {"and": [
+                    "SpaceJump",
+                    "canUsePowerBombs",
```

Comment:
Are the power bombs still needed for either of the traversal options, btw?

Reply:
If you're asking if PB's are needed to dig into the statue after the first time, yes. I wasn't sure how to do about this one but figured it was best to keep it in both at unlock and link.

Reply:
Ok. It wasn't my intent, but I don't suppose it matters much either way.